ambassador/tests/kat/test_ambassador.py

```diff
@@ -463,54 +439,6 @@ def queries(self):
         yield Query(self.url("missme/"), expected=404)
         yield Query(self.url("missme-array/"), expected=404)
 
-class AuthenticationTest(AmbassadorTest):
```

Comment:
We still need to test the v0 `AuthService`, sadly.

Reply:
No worries, thanks for letting me know about V0 support. I will make sure to have this test back.

Reply:
Can't have them running with V1. See comment below please.